### DISCUSSION TITLE
Coordination for Next Release of `review-web` and `review-database`

### DISCUSSION THREAD

**Discussion by msk** [aicers/review-web#290] [Q&A] [answered]
@petabi is maintaining an application that depends on both review-web and review-database. It’s been quite some time since our last release, and over the past few months, review-database has accumulated several database migrations. We’re eager to release our application soon to avoid prolonged migration times.

However, since our release depends on the next version of review-web, I wanted to check in and see if you have an estimated timeline for your next release. Additionally, it would be really helpful if you could share any specific features or changes in review-database that your team requires in the upcoming version. This way, we can ensure those updates are included, facilitating a smooth release for both crates.

Cc: @minshao

**Comment by sehkone** [comment, marked as answer]
It would be great if the next release of review-web includes the features implemented in #255 and #280. I expect #280 will be merged in a couple of days, but I need to check how long #255 will take. If #255 seems like it will take many days, it might be better to release separate versions in sequence.

**Reply by msk** [reply]
Thanks for the update. 

Then, on the review-database side, petabi/review-database#315 is the only remaining prerequisite for review-web to be release-ready. We're going to release a new version of review-database once petabi/review-database#316 has been merged, which will address the necessary changes outlined in petabi/review-database#315.

Let me know if there's anything else we need to consider.

**Comment by sehkone** [comment]
@msk, the new release of review-web only requires #292.

**Reply by msk** [reply]
Thanks for the update. We've just released review-database-0.30.0. Please update review-web's dependency accordingly when you release the new version.
